sqlite_helpers.py

```diff
@@ -0,0 +1,107 @@
+import sqlite3
+import random
+import string
+from fastapi import FastAPI, Request, HTTPException
+from fastapi.responses import RedirectResponse, HTMLResponse
+from datetime import datetime, timedelta
+
+def maybe_create_table(sqlite_file: str) -> bool:
+    db = sqlite3.connect(sqlite_file)
+    cursor = db.cursor()
+
+    try :
+        createTableQuery = "CREATE TABLE IF NOT EXISTS urls (id INTEGER PRIMARY KEY, url TEXT, alias TEXT, timestamp DATETIME)"
```

Comment:
python best practices use snake case, lets change all instances of camelCase to snake_case

Reply:
done